GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MoveRenameFolderTests.cs

```diff
@@ -181,14 +174,12 @@ public void MoveAndRenameUnhydratedFolderToNewFolder()
             this.Enlistment.GetVirtualPathTo(Path.Combine(movedFolderPath, testFileName)).ShouldBeAFile(this.fileSystem).WithContents(TestFileContents);
         }
 
-        // This test requires expansion on PreDelete to be implemented
         // MacOnly because renames of partial folders are blocked on Windows
         [TestCase]
-        [Category(Categories.MacTODO.M2)]
         [Category(Categories.MacOnly)]
         public void MoveFolderWithUnhydratedAndFullContents()
         {
-            string testFileName = "MoveFolderWithUnhydratedAndFullContents.cs";
+            string testFileName = "MoveFolderWithUnhydratedAndFullContents.cpp";

```

Comment:
Why was this changed?

Reply:
The old path was just wrong (see [MoveFolderWithUnhydratedAndFullContents.cpp](https://gvfs.visualstudio.com/ci/_git/ForTests?path=%2FTest_EPF_MoveRenameFolderTests%2FMoveFolderWithUnhydratedAndFullContents%2FMoveFolderWithUnhydratedAndFullContents.cpp&version=GBFunctionalTests%2F20180214)).

This hasn't caused any failures up until now as the tests have been disabled on Windows for a very long time.